docs/source/configurator.rst

```diff
@@ -125,6 +125,8 @@ Node type dependent options for [nodeN] : ::
 
    type="distributor" : No options
 
+   type="denester" : No options
```

Comment:
It needs to have a `sequence_identifier` option for specifying the sequence identifier of the output sequence, just like the Resequencer has.